pkg/hubble/peer/types/client.go

```diff
@@ -85,9 +81,7 @@ func (b RemoteClientBuilder) Client(target string) (Client, error) {
 		})
 		opts = append(opts, grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)))
 	}
-	ctx, cancel := context.WithTimeout(context.Background(), b.DialTimeout)
```

Comment:
Same here for the `RemoteClientBuilder`. The DialTimeout is now unused and can be removed